packages/legacy-client/src/client/actions/actions.test.ts

```diff
@@ -116,6 +118,16 @@ describe("Actions", () => {
           ]
         >();
 
+      expectTypeOf<Parameters<typeof bulkActions.createTask>>()
```

Comment:
Added these just to make sure `BulkAction` types and `BatchAction` types are equivalent